pkg/data/schema.go

```diff
@@ -68,6 +68,16 @@ create table if not exists completed_movements
 
     scenario_run_id integer not null references scenario_runs (id)
 );
+
+create table if not exists cpu_utilizations
+(
+	id 					integer primary key,
+	cpu_utilization 	real 					not null,
```

Comment:
Tabs vs spaces again.

Reply:
Set up my IDE to avoid this things. See e91267565b8ba92593c89fe969f1d65a35e99baf commit.